rules/windows/process_creation/win_mal_adwind.yml

```diff
@@ -23,25 +23,25 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine:
-            - '*\AppData\Roaming\Oracle*\java*.exe *'
-            - '*cscript.exe *Retrive*.vbs *'
+        CommandLine|contains:
+            - '\AppData\Roaming\Oracle*\java*.exe '
+            - 'cscript.exe *Retrive*.vbs '
 ---
 logsource:
     product: windows
     service: sysmon
 detection:
     selection:
         EventID: 11
-        TargetFilename:
-            - '*\AppData\Roaming\Oracle\bin\java*.exe'
-            - '*\Retrive*.vbs'
+        TargetFilename|endswith:
+            - '\AppData\Roaming\Oracle\bin\java*.exe'
+            - '\Retrive*.vbs'
```

Comment:
could you please work out the logic to get rid of asterisks here?

Reply:
```
detection:
    selection:
        - CommandLine|contains|all:
            - '\AppData\Roaming\Oracle'
            - '\java'
            - '.exe '
        - CommandLine|contains|all:
            - 'cscript.exe'
            - 'Retrive'
            - '.vbs '
```
```
detection:
    selection:
        EventID: 11
        TargetFilename|contains:
            - '\AppData\Roaming\Oracle\bin\java'
            - '\Retrive'
        TargetFilename|endswith:
            - '.exe'
            - '.vbs'
```
What do you think?

Reply:
I think it's better this way:

```
---
logsource:
    category: process_creation
    product: windows
detection:
    selection:
        - CommandLine|contains|all:
            - '\AppData\Roaming\Oracle'
            - '\java'
            - '.exe '
        - CommandLine|contains|all:
            - 'cscript.exe'
            - 'Retrive'
            - '.vbs '
---
logsource:
    category: file_event
    product: windows
detection:
    selection:
        - TargetFilename|contains|all:
            - '\AppData\Roaming\Oracle\bin\java'
            - '.exe'
        - TargetFilename|contains|all:
            - '\Retrive'
            - '.vbs'
---
logsource:
    category: registry_event
    product: windows
detection:
    selection:
        TargetObject|startswith: HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Run
        Details|startswith: '%AppData%\Roaming\Oracle\bin\\'
```

Reply:
fixed